### PR TITLE
social_register_hepler_msg -> social_register_helper_msg

### PR DESCRIPTION
**Pull request by strk**

### CODE REVIEW

**Review by lunny — COMMENTED** [review]
LGTM

File: options/locale/locale_bg-BG.ini [hunk 1]
```diff
@@ -142,7 +142,7 @@ search=Търсене
 [auth]
 create_new_account=Създай нов профил
 register_hepler_msg=Вече имате профил? Впишете се сега!
```
Comment (ethantkoenig) [inline options/locale/locale_bg-BG.ini]: @strk While you're at it, would you mind fixing this one too?

**Review by appleboy — APPROVED** [review]

**Comment by Bwko** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Pull request merged**